client/src/components/RenderToken.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+// planning to visualise tokens in future so it is separate component
+interface IRenderTokenProps {
+	contractAddress: string;
+}
+
+export const RenderToken = ({ contractAddress }: IRenderTokenProps) => {
```

Comment:
RenderToken is a poor naming convention, should just be Token or similar. We could add Render before every component technically